ros/src/computing/perception/detection/lib/image/librcnn/CMakeLists.txt

```diff
@@ -0,0 +1,65 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(librcnn)
+
+find_package(catkin REQUIRED COMPONENTS
+  roscpp
+  sensor_msgs
+  std_msgs
+)
+
+find_package(OpenCV REQUIRED)
+
+###########################################CAFFE NEEDS TO BE PREVIOUSLY COMPILED####################
+##############DONT FORGET TO INSTALL fastrcnn's caffe dynamic libraries to /usr/local/lib	#### 
+set(CAFFE_PATH "$ENV{HOME}/fast-rcnn/caffe-fast-rcnn/distribute")				####
+set(CUDA_INCLUDE_PATH "/usr/local/cuda/include/")						####
+####################################################################################################
+
+if(EXISTS "${CAFFE_PATH}")
+
+###################################
+## catkin specific configuration ##
+###################################
+catkin_package(
+  INCLUDE_DIRS include
+  LIBRARIES librcnn
+)
+
+###########
+## Build ##
+###########
+
+set(CMAKE_CXX_FLAGS "-std=c++0x -O2 -Wall ${CMAKE_CXX_FLAGS}")
```

Comment:
Please use `-std=c++11` option instead of `-std=c++0x` because we need not to support older compiler now.